### PR TITLE
Feat/pro 4252/rocko

### PR DESCRIPTION
**Pull request by OYTIS**
I'll create a PR to updater-repo in order to test rocko/master branches.

### CODE REVIEW

File: conf/include/bblayers/sota.inc [hunk 1]
```diff
@@ -1,4 +1,6 @@
-
+BBLAYERS += "${METADIR}/poky/meta"
+BBLAYERS += "${METADIR}/poky/meta-poky"
+BBLAYERS += "${METADIR}/poky/meta-yocto-bsp"
```
Comment (patrickvacek) [inline conf/include/bblayers/sota.inc]: This seems to cause my new bblayers.conf to have duplicated entries for each of these three layers. Do we need this?

Comment (OYTIS) [reply conf/include/bblayers/sota.inc]: Um... no, I was sure I've deleted that.

Comment (OYTIS) [reply conf/include/bblayers/sota.inc]: Just fixed these, thanks. There are some minor fixes coming for RPi3 and maybe Minnowboard, but I'm not sure if it's worth adding them to this PR.

**Comment by patrickvacek** [comment]
I'm stuck on bitbake errors like the following:
```
ERROR: gettext-minimal-native-0.19.8.1-r0 do_patch: Command Error: 'quilt --quiltrc /home/patrick/Code/qemu_rocko/build/tmp/work/x86_64-linux/gettext-minimal-native/0.19.8.1-r0/recipe-sysroot-native/etc/quiltrc push' exited with 0  Output:
/bin/sh: 1: quilt: not found
```
I cannot figure out why it can't find quilt. That's pretty basic.

**Comment by OYTIS** [comment]
Never seen that, looks like the environment is fundamentally broken. I think quilt should be pulled by some bbclasses belonging to oe-core on early stages. Do you use updater-repo to pull everything together?

**Comment by patrickvacek** [comment]
Yeah, I'm pretty confused. This was a new build directory, and yes, I'm using updater-repo.

**Pull request merged**